client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -619,7 +620,7 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 
 const mapStateToProps = (
 	state: T.AppState,
-	{ postId, postType, duplicatePostId, fseParentPageId, creatingNewHomepage }: Props
+	{ postId, postType, duplicatePostId, fseParentPageId, creatingNewHomepage, editorType }: Props
```

Comment:
```suggestion
	{ postId, postType, duplicatePostId, fseParentPageId, creatingNewHomepage, editorType = 'post' }: Props
```

might be nice to add a default value so it's not undefined in the normal case